README.md

```diff
@@ -160,6 +160,10 @@ final Atlas atlas2;
 new PackedAtlasCloner().cloneFrom(new MultiAtlas(atlas1, atlas2)).save(new File("/path/to/file.atlas"));
 ```
 
+# PyAtlas
+
+`Atlas` features a lightweight Python version of the core functionality. The `pyatlas` is available as a Python package from PyPI, but can also be built and installed locally. Check out the `pyatlas` subfolder and the `pyatlas/README.md` file for more information!
```

Comment:
You could link it directly here using `... [Readme file](pyatlas/README.md) ...`

Reply:
Ah ok let me add that. I did not realize you could do relative URLs.